src/main/java/org/radarcns/redcap/util/FieldParser.java

```diff
@@ -0,0 +1,6 @@
+package org.radarcns.redcap.util;
+
+public interface FieldParser<T> {
+
+    T parseField(Object fieldValue);
```

Comment:
You can make the field value field generic too, like -
```java
public interface FieldParser<T, R> {
    T parseField(R fieldValue);
}
```

And in case of `AttributeFieldParser`, it will implement `FieldParser<String, String>`. This improves typing as compared to the very generic Object class.